news/_posts/2020-10-01-hacktoberfest-2020-is-coming.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "Hacktoberfest 2020 is coming"
+subtitle: "Bring the lights on open source projects"
+date:  2020-10-01 09:10:11
+authors: [LouiseBonnard]
+icon: icon-github-alt
+image: /assets/images/2020/09/Hacktoberfest-2020.png
```

Comment:
Image is maybe a little bit bigger than needed 😁

Reply:
Oops, you're right, I'll create a banner to have it better sized, thanks for the comment! :-)

Reply:
@matks, I've just changed the image, I let you check!